registration-processor/pre-processor/registration-processor-packet-classifier-stage/src/main/java/io/mosip/registration/processor/stages/packetclassifier/utility/PacketClassifierUtility.java

```diff
@@ -0,0 +1,60 @@
+package io.mosip.registration.processor.stages.packetclassifier.utility;
+
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import io.mosip.kernel.core.exception.BaseCheckedException;
+import io.mosip.registration.processor.core.exception.util.PlatformErrorMessages;
+
+@Component
+public class PacketClassifierUtility {
+
+	/**
+	 * The mandatory languages that should be used when dealing with field type that
+	 * has values in multiple languages
+	 */
+	@Value("#{'${mosip.mandatory-languages}'.split(',')}")
+	private List<String> mandatoryLanguages;
+
+	/**
+	 * The optional languages that should be used when dealing with field type that
+	 * has values in multiple languages
+	 */
+	@Value("#{'${mosip.optional-languages}'.split(',')}")
+	private List<String> optionalLanguages;
+
+	/** The constant for language label in JSON parsing */
+	private static final String LANGUAGE_LABEL = "language";
+
+	public String getTagLanguageBasedOnFieldDTO(String value) throws JSONException, BaseCheckedException {
+		if (!mandatoryLanguages.isEmpty()) {
+			for (String language : mandatoryLanguages) {
```

Comment:
For manditory language, you don't need a for loop, since all the value will be available in all the mandatory language, check for mandatoryLanguages.get(0), if data is not available throw an error.

Reply:
changed to checking for first mandatory language only...